### PR TITLE
Fixes for ZoneHVAC:HybridUnitaryHVAC

### LINKED ISSUE #7505: ZoneHVAC:HybridUnitaryHVAC Availability Manager List Name
Issue overview
--------------
The Availability Manager List Name input has no effect on the operation of the hybrid unitary system.  The field input is referenced [here](https://github.com/NREL/EnergyPlus/blob/81001c598a82ad3474a838c24225ec0f5b0e3ed6/src/EnergyPlus/HybridUnitaryAirConditioners.cc#L486).

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.1.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7506: ZoneHVAC:HybridUnitaryHVAC Requested Outdoor Air Flow Rate
Issue overview
--------------
The [Zone Hybrid Unitary HVAC Requested Outdoor Air Ventilation Mass Flow Rate](https://bigladdersoftware.com/epx/docs/9-1/input-output-reference/group-zone-forced-air-units.html#zone-hybrid-unitary-hvac-requested-outdoor-air-ventilation-mass-flow-rate-kgs) output and the system node mass flow rate for the outdoor air node show an output of 0 when the model specifies outdoor air requirements using the [Design Specification Outdoor Air Object Name](https://bigladdersoftware.com/epx/docs/9-1/input-output-reference/group-zone-forced-air-units.html#field-design-specification-outdoor-air-object-name) input field.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.1.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7520: ZoneHVAC:HybridUnitaryHVAC Second and Third fuel types - consumption is not metered
Issue overview
--------------
Based on code review while removing fuel type synonyms, it appears that:

1. These energy consumption variables for ZoneHVAC:HybridUnitaryHVAC are not connected to a meter:
Zone Hybrid Unitary HVAC Secondary Fuel Consumption
Zone Hybrid Unitary HVAC Third Fuel Consumption Rate

2. Zone Hybrid Unitary HVAC Supply Fan Electric Energy is metered, that seems ok. But Zone Hybrid Unitary HVAC Electric Energy is also metered. If the fan energy is included in this, then it's gettting double-counted on the electric meter.

3. Field: First Fuel Type has a full range of choices, but the default it electricity (in code, noted in the IDD), but this appears to be connected to the Electric meter - not variable based on which fuel type is specified in the input. Actually, I'm not finding where this is metered.

4. There are several (maybe more) variables which have SetupOutputVariable statements but searching HybridUnitaryAirConditioners.cc for the variable name shows it does not appear to be used. Stopped looking after these three, there could be more?
```
            SetupOutputVariable("Zone Hybrid Unitary HVAC Electric Energy",
                                OutputProcessor::Unit::J,
                                ZoneHybridUnitaryAirConditioner(UnitLoop).FinalElectricalEnergy,

            SetupOutputVariable("Zone Hybrid Unitary HVAC Requested Outdoor Air Ventilation Mass Flow Rate",
                                OutputProcessor::Unit::kg_s,
                                ZoneHybridUnitaryAirConditioner(UnitLoop).MinOA_Msa,
            SetupOutputVariable("Zone Hybrid Unitary HVAC Ventilation Air Mass Flow Rate",
                                OutputProcessor::Unit::kg_s,
                                ZoneHybridUnitaryAirConditioner(UnitLoop).SupplyVentilationAir,
```

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus v9.2.0-https://github.com/NREL/EnergyPlus/commit/60ed7207c2f3058fe8dc400a1aa741149094bebe
 - Noticed while working on #6601
### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here) - don't have any defect files - will need some for this
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7749: ZoneHVAC:HybridUnitaryHVAC load not being accounted for in additional HVAC equipment
Issue overview
--------------
The ZoneHVAC:HybridUnitaryHVAC object is not included in the list of
[NumZoneHVACTerminalTypes](https://github.com/NREL/EnergyPlus/blob/b1c11ff5a75e8e34a6e667b25b8d350f4f1cddae/src/EnergyPlus/DataHVACGlobals.cc#L508) and its corresponding array.  It is also missing from the [ReportMaxVentilationLoads](https://github.com/NREL/EnergyPlus/blob/b1c11ff5a75e8e34a6e667b25b8d350f4f1cddae/src/EnergyPlus/SystemReports.cc#L4342) function that calculates and reports the zone ventilation loads.  This is preventing any cooling and/or ventilation the unit supplies to zone to be passed on to subsequent equipment.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7750: ZoneHVAC:HybridUnitaryHVAC - Change lookup tables to use Watts instead of kW
Issue overview
--------------
The lookup tables for all fuel power and consumption rates in the ZoneHVAC:HybridUnitaryHVAC object currently expect table values to be in kW, instead of the EnergyPlus typical standard of Watts.  This is also not explained anywhere in the documentation or IDD file.  The calculation from the output value from the table to the EnergyPlus output can be seen [here](https://github.com/NREL/EnergyPlus/blob/b1c11ff5a75e8e34a6e667b25b8d350f4f1cddae/src/EnergyPlus/HybridEvapCoolingModel.cc#L2128).

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7751: ZoneHVAC:HybridUnitaryHVAC - Missing output variables for return air and relief air values
Issue overview
--------------
The ZoneHVAC:HybridUnitaryHVAC object doesn't output the flow rate for the return air stream to the unit or the temperature, flow rate, or humidity for the relief air stream.  The output variables for the ZoneHVAC:HybridUnitaryHVAC object can be found [here](https://github.com/NREL/EnergyPlus/blob/b1c11ff5a75e8e34a6e667b25b8d350f4f1cddae/src/EnergyPlus/HybridUnitaryAirConditioners.cc#L595).

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7752: ZoneHVAC:HybridUnitaryHVAC - Output variable for outdoor air node is reporting no flow rate during 100% OA operation
Issue overview
--------------
When the ZoneHVAC:HybridUnitaryHVAC unit is running, the output shows supply air flow and cooling operation but the outdoor air node has an output of 0 for the flow rate at 100% outdoor air operation.  Seems to be using the wrong parameter to grab the minimum outside air in the [DetermineCoolingVentilationOrHumidificationNeeds](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/HybridEvapCoolingModel.cc#L1874) function.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7753: ZoneHVAC:HybridUnitaryHVAC is not included in any reporting summary tables for output
Issue overview
--------------
The ZoneHVAC:HybridUnitaryHVAC object is not included in any output summary tables showing information regarding the object parameters.  Should at least include maximum supply air flow to show the resulting size of the unit based on the System Maximum Supply Air Volume Flow Rate input and Scaling Factor Input for the object.  Everything else is dependent on the lookup tables.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by matthew-larson**
Pull request overview
---------------------
 - Fixes #7505, #7506, #7520, #7748, #7749, #7750, #7751, #7752, #7753, #7754, #7800 
 - This pull request fixes various issues regarding the ZoneHVAC:HybridUnitaryHVAC object including correct load and ventilation calculations, handling of minimum ventilation requirements, use of availability managers, and correct outputs for variables, meters, and summary tables.  It also changes the lookup tables for System Electric Power, Supply Fan Electric Power, Second Fuel Consumption, and Third Fuel Consumption to expect values to be in Watts instead of kW. The associated test file UnitaryHybridUnitTest_DOSA.idf and example file UnitaryHybridAC_DedicatedOutsideAir.idf was also updated to reflect lookup tables in Watts instead of kW.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by nealkruis** [comment]
Note: Currently inculdes changes proposed in PR #7528.

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -39847,41 +40047,49 @@ ZoneHVAC:HybridUnitaryHVAC,
...
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the System Third Fuel Consumption Lookup Table for Mode 25.
+        \note Units for lookup table values should be in W.
         \note If this field is blank, Mode 25 does not consume a third fuel.
   A250, \field Mode 25 System Water Use Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the System Water Use Lookup Table for Mode 25.
+        \note Units for lookup table values should be in kg/s.
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:40092]: The number of lookup table fields here is quite ridiculous.  But adding these notes are a good addition that doesn't affect functionality at all, and can only help people get the right inputs in there.

File: src/EnergyPlus/HybridEvapCoolingModel.cc [hunk 2]
```diff
@@ -1425,16 +1423,16 @@ namespace HybridEvapCoolingModel {
             return -1;
         } // because it should be fractional, this should only really be possible if its called from a unit test
 
-        Real64 Wosa = PsyWFnTdbRhPb(StepIns.Tosa, StepIns.RHosa, 101325);
-        Real64 Wra = PsyWFnTdbRhPb(StepIns.Tra, StepIns.RHra, 101325);
+        Real64 Wosa = PsyWFnTdbRhPb(StepIns.Tosa, StepIns.RHosa, OutBaroPress);
```
Comment (Myoldmopar) [inline src/EnergyPlus/HybridEvapCoolingModel.cc:1426]: 👍

**Review by Myoldmopar — APPROVED** [review]
I'm not going to slog through every defect this fixes.  It looks like this is going to make some great improvements in the hybrid unitary system, and CI is very happy, only showing diffs in the one expected file.

**Pull request merged**